### PR TITLE
Update guide with test case for latest syntaxes

### PR DESCRIPTION
**Pull request by ldclakmal**
## Purpose
- Fix syntax issues
- Fix test cases
- Update readme file for latest syntaxes

### CODE REVIEW

**Review by ayomawdb — CHANGES_REQUESTED** [review]
Added few comments relevant to the usage of `untaint` expression. Other than that, the changes look fine and suggestions will surely improve the guide.

File: README.md [hunk 1]
```diff
@@ -61,42 +61,47 @@ secure-restful-service
...
-        string orderId = orderReq.Order.ID.toString() but { () => "" };
+        string orderId = orderReq.Order.ID.toString();
         ordersMap[orderId] = orderReq;
 
         // Create response message.
-        json payload = {status:"Order Created.", orderId:orderId};
+        json payload = { status: "Order Created.", orderId: orderId };
         http:Response response;
-        response.setJsonPayload(payload);
+        response.setJsonPayload(untaint payload);
```
[the remark this answers is not in the document]
Comment (ldclakmal) [reply README.md]: Yes, agree. Fixed accordingly.

File: README.md [hunk 2]
```diff
@@ -134,18 +139,18 @@ service<http:Service> order_mgt bind listener {
 
         http:Response response;
         // Set the JSON payload to the outgoing response message to the client.
-        response.setJsonPayload(existingOrder);
+        response.setJsonPayload(untaint existingOrder);
```
[the remark this answers is not in the document]
Comment (ldclakmal) [reply README.md]: Fixed.

File: README.md [hunk 3]
```diff
@@ -155,34 +160,37 @@ service<http:Service> order_mgt bind listener {
...
+        } else {
+            response.statusCode = 404;
             payload = "Order : " + orderId + " cannot be found.";
         }
-    
+
         // Set the JSON payload in the outgoing response message.
-        response.setJsonPayload(payload);
-    
+        response.setJsonPayload(untaint payload);
```
[the remark this answers is not in the document]
Comment (ldclakmal) [reply README.md]: Fixed.

File: guide/secure_restful_service/secure_order_mgt_service.bal [hunk 4]
```diff
@@ -60,7 +61,7 @@ service<http:Service> order_mgt bind listener {
         ordersMap[orderId] = orderReq;
 
         // Create response message.
-        json payload = {status:"Order Created.", orderId:orderId};
+        json payload = { status: "Order Created.", orderId: orderId };
         http:Response response;
         response.setJsonPayload(untaint payload);
```
[the remark this answers is not in the document]
Comment (ldclakmal) [reply guide/secure_restful_service/secure_order_mgt_service.bal]: Fixed.

File: guide/secure_restful_service/secure_order_mgt_service.bal [hunk 5]
```diff
@@ -102,16 +103,16 @@ service<http:Service> order_mgt bind listener {
         // Set the JSON payload to the outgoing response message to the client.
         response.setJsonPayload(untaint existingOrder);
```
[the remark this answers is not in the document]
Comment (ldclakmal) [reply guide/secure_restful_service/secure_order_mgt_service.bal]: Fixed.

File: guide/secure_restful_service/secure_order_mgt_service.bal [hunk 6]
```diff
@@ -124,15 +125,15 @@ service<http:Service> order_mgt bind listener {
         response.setJsonPayload(untaint payload);
```
[the remark this answers is not in the document]
Comment (ldclakmal) [reply guide/secure_restful_service/secure_order_mgt_service.bal]: Fixed.

File: guide/secure_restful_service/secure_order_mgt_service.bal [hunk 7]
```diff
@@ -151,6 +152,6 @@ service<http:Service> order_mgt bind listener {
...
```
[the remark this answers is not in the document]
Comment (ldclakmal) [reply guide/secure_restful_service/secure_order_mgt_service.bal]: Fixed.

File: README.md [hunk 8]
```diff
@@ -155,34 +160,37 @@ service<http:Service> order_mgt bind listener {
 
         json payload = "Order : " + orderId + " removed.";
         // Set a generated payload with order status.
-        response.setJsonPayload(payload);
+        response.setJsonPayload(untaint payload);
```
[the remark this answers is not in the document]
Comment (ldclakmal) [reply README.md]: Fixed.

**Review by ayomawdb — APPROVED** [review]
Verified new changes and approved.

File: README.md [hunk 9]
```diff
@@ -619,40 +669,43 @@ Ballerina has a log package for logging to the console. You can import ballerina
...
-       beats {
-	       port => 5044
-	      }  
+    input {
+    beats {
+       port => 5044
+      }
+    }
+
+    filter {
```
Comment (VijithaEkanayake) [inline README.md]: Formatting issue

File: guide/secure_restful_service/secure_order_mgt_service.bal [hunk 10]
```diff
@@ -26,43 +24,53 @@ import ballerinax/kubernetes;
...
+        orderId = getUntaintedStringIfValid(orderId);
+
         ordersMap[orderId] = orderReq;
 
         // Create response message.
-        json payload = {status:"Order Created.", orderId:orderId};
+        json payload = { status: "Order Created.", orderId: orderId };
         http:Response response;
-        response.setJsonPayload(untaint payload);
+        response.setJsonPayload(payload);
```
Comment (VijithaEkanayake) [inline guide/secure_restful_service/secure_order_mgt_service.bal]: Let's use setPayload() here and any other applicable places

File: README.md [hunk 11]
```diff
@@ -136,56 +156,85 @@ service<http:Service> order_mgt bind listener {
...
+
         // Find the requested order from the map and retrieve it in JSON format.
-        json? payload = ordersMap[orderId];
         http:Response response;
-        if (payload == null) {
+        json payload;
+        if (ordersMap.hasKey(orderId)) {
+            payload = ordersMap[orderId];
+        } else {
+            response.statusCode = 404;
```
Comment (VijithaEkanayake) [inline README.md]: Let's use the constants available in Ballerina for status codes

File: README.md (hunk 9 above)
Comment (ldclakmal) [reply README.md]: Fixed.

File: guide/secure_restful_service/secure_order_mgt_service.bal (hunk 10 above)
Comment (ldclakmal) [reply guide/secure_restful_service/secure_order_mgt_service.bal]: Fixed.

File: README.md (hunk 11 above)
Comment (ldclakmal) [reply README.md]: Fixed.

**Review by VijithaEkanayake-zz — APPROVED** [review]

**Pull request merged**